### PR TITLE
update mkdocs-material version (#415)

### LINKED ISSUE #415: `make docs-serve`/mkdocs builds failing because of jinja2 error
### Bug Description

The newest version of jinja is incompatible with mkdocs. Building locally and also via github actions on merges to main currently breaks.

#### Steps to Reproduce

1. pull main
2. run `make docs-serve`
3. get the following error:
``` 
AttributeError: module 'jinja2' has no attribute 'contextfilter' 
```

### Expected behavior

Build the docs locally, and via github actions on merges 

### Environment

* Version:
* OS:
* Python Version:
* Docker Version:

### Additional context
Mkdocs pushed [a patch](https://github.com/squidfunk/mkdocs-material/releases/tag/8.2.7) to limit the jinja version range. Updating the pinned version in `requirements.txt` should resolve the issue for now, but might not be the best strategy for versioning in the future.

### PR DESCRIPTION
**Pull request by conceptualshark**
Closes #415

### Code Changes

* update mkdocs version to be compatible with recent jinja2 release 

### Description Of Changes

Mkdocs pushed [a patch](https://github.com/squidfunk/mkdocs-material/releases/tag/8.2.7) to limit the jinja version range. Updating the pinned version in the docs' `requirements.txt` will resolve the issue for now, but might not be the best strategy for versioning in the future.

### CODE REVIEW

**Comment by ThomasLaPiana** [comment]
@conceptualshark is this ready for review?

If it's ready go ahead and assign reviewers so we make sure it doesn't get forgotten about! If it still needs some work please change it to draft status

Thank you for the fix!

File: docs/fides/docs/schemas/config_schema.json [hunk 1]
```diff
@@ -29,7 +29,8 @@
       "title": "Cli",
       "default": {
         "local_mode": false,
-        "server_url": "http://localhost:8080"
+        "server_url": "http://localhost:8080",
+        "analytics_id": "6044ce70af305cf78969a3cd7a5371ba"
```
Comment (PSalant726) [inline docs/fides/docs/schemas/config_schema.json:33]: Please remove this line. Even though an `analytics_id` is required (for clients that are opted into analytics), this value is associated with a specific instance and should not exist in VCS.

File: docs/fides/docs/schemas/config_schema.json [hunk 2]
```diff
@@ -185,6 +187,14 @@
             "fidesctl__cli__server_url"
           ],
           "type": "string"
+        },
+        "analytics_id": {
+          "title": "Analytics Id",
+          "default": "6044ce70af305cf78969a3cd7a5371ba",
```
Comment (PSalant726) [inline docs/fides/docs/schemas/config_schema.json:193]: This default should be `""`, or removed.

**Review by PSalant726 — CHANGES_REQUESTED** [review]

**Review by PSalant726 — CHANGES_REQUESTED** [review]
To clarify my previous comments, the `analytics_id` and `analytics_opt_out` config options are valid (new) config options. Ideally they wouldn't be removed completely from the schema. It's just the _value_ of the `analytics_id` that needs to be omitted, as it represents a unique instance of fidesctl.

**Review by PSalant726 — APPROVED** [review]

**Review by ThomasLaPiana — APPROVED** [review]
LGTM! Thanks for the fix

**Pull request merged**